src/main/java/htsjdk/samtools/cram/build/ContainerFactory.java

```diff
@@ -58,8 +64,7 @@ public Container buildContainer(final List<CramCompressionRecord> records) {
             slices.add(slice);
         }
 
-        final Container container = Container.initializeFromSlices(slices);
-        container.compressionHeader = compressionHeader;
```

Comment:
I don't recall why I moved this inside the call to `initializeFromSlices()`.  I can take it back out - they seem pretty equal to me.